src/comicgen.js

```diff
@@ -54,6 +92,52 @@ export default function comicgen(selector, options) {
   })
 }
 
+
+function create_parametric_svg(node, param) {
```

Comment:
Please simplify like crazy. Intepolating 2 SVGs ought to be trivial. Re-use code. This ideally should be one function - preferably from some existing library - that interpolates between 2 SVGs. Find such a library if required.